### DISCUSSION TITLE
SSI-KIT: EBSI accredating and onboarding a trusted issuer

### DISCUSSION THREAD

**Discussion by soufianeAmaador** [walt-id/.github#32] [Help] [answered]
Hello,

I am currently using the SSI-kit as a direct jvm dependency to create use cases for EBSI.
Does SSI-kit have methods that allows for accrediting and onboarding a Legal Entity (LE) to become a Trusted Issuer (TI)?
The signatory interface currently only allows me to issue verifiable credentials (VC), list all the templates, etc.

**Comment by philpotisk** [comment, marked as answer]
Hi @soufianeAmaador , we would like to support inserting & updating records in these registries via the SSI Kit directly, but until now this only could be done via a manual support request to the EBIS team. Once this process changes, we would surely implement this.

BTW: We finalized the transition from the EBSI pre-prod to the pilot system now. So at least the DID registry from the pilot-system can be used now.

**Reply by soufianeAmaador** [reply]
Thank you for the swift reply! I'll try to contact the EBSI help desk.
